diff --git a/faciallandmarks/pluginv2/CMakeLists.txt b/faciallandmarks/pluginv2/CMakeLists.txt
index 5970d8b..857c178 100644
--- a/faciallandmarks/pluginv2/CMakeLists.txt
+++ b/faciallandmarks/pluginv2/CMakeLists.txt
@@ -5,7 +5,7 @@ cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 project(FlnetSoftargmax LANGUAGES CXX)
 
 # Enable all compile warnings
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Werror")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long")
 # Use C++11
 set (CMAKE_CXX_STANDARD 11)
 
